obspy/io/ah/core.py

```diff
@@ -113,7 +113,13 @@ def _get_ah_version(filename):
 
 
 def _unpack_string(data):
-    return data.unpack_string().split(b'\x00', 1)[0].strip().decode("utf-8")
+    string = data.unpack_string()
+    try:
+        string = string.split(b'\x00', 1)
```

Comment:
I don't see how this would ever be able to trigger a `UnicodeDecodeError`.. this is on bytes? what was the original problem?

Would be good to add a small test case I think..